index.html

```diff
@@ -1,60 +1,324 @@
         </div>
-    </div>
-
-</body>
+        <div class="about-me1-separation">
+          <div class="line2"></div>
+        </div>
+        <div class="about-me-2">
+          <div class="grid1">
+            <div class="topic">
+              <img src="./svg/language.svg" alt="" />
+              <p>Language</p>
+            </div>
+            <ul class="discussion">
+              <li>
+                <a href="https://en.wikipedia.org/wiki/JavaScript"
+                  >Javascript</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://en.wikipedia.org/wiki/Ruby_(programming_language)"
+                  >Ruby</a
+                >
+              </li>
+              <li><a href="https://en.wikipedia.org/wiki/HTML">html</a></li>
+              <li><a href="https://en.wikipedia.org/wiki/CSS">css</a></li>
+            </ul>
+          </div>
+          <div class="grid2">
+            <div class="topic">
+              <img src="./svg/frameworks.svg" alt="" />
+              <p>Frameworks</p>
+            </div>
+            <ul class="discussion">
+              <li>
+                <a
+                  href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a href="https://en.wikipedia.org/wiki/Ruby_on_Rails"
+                  >Ruby On Rails</a
+                >
+              </li>
+              <li><a href="https://en.wikipedia.org/wiki/RSpec">RSPec</a></li>
+              <li>
+                <a href="https://en.wikipedia.org/wiki/Capybara_(software)"
+                  >CapyBara</a
+                >
+              </li>
+              <li>
+                <a href="https://en.wikipedia.org/wiki/Selenium_(software)"
+                  >Selenium</a
+                >
+              </li>
+            </ul>
+          </div>
+          <div class="grid3">
+            <div class="topic">
+              <img src="./svg/skills.svg" alt="" />
+              <p>Skills</p>
+            </div>
+            <ul class="discussion">
+              <li>
+                <a href="https://en.wikipedia.org/wiki/CodeLite">CodeKit</a>
+              </li>
+              <li><a href="https://en.wikipedia.org/wiki/GitHub">Github</a></li>
+              <li>
+                <a href="https://en.wikipedia.org/wiki/CodePen">Codepen</a>
+              </li>
+              <li><a href="https://en.wikipedia.org/wiki/GitLab">GitLab</a></li>
+              <li>
+                <a href="https://en.wikipedia.org/wiki/Linux_console"
+                  >Terminal</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <div class="form-section" id="contact">
+        <a href="#form"></a>
+        <div class="form-grid">
+          <h2>I'm always interested in hearing about new projects.</h2>
```

Comment:
This content is missing a portion of the text from the template design. Please fix this as it is a project requirement.

<img width="895" alt="Screen Shot 2022-06-08 at 8 42 35" src="https://user-images.githubusercontent.com/82426682/172631640-864098ee-4071-4274-bf34-504da975b34c.png">